### PR TITLE
Es6: bootstraps/enhanced/gallery

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

es6

### CODE REVIEW

File: static/src/javascripts/bootstraps/enhanced/gallery.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+const transcludeMostPopular = (): void => {
+    const mostViewed = new Component();
+    const container = qwery('.js-gallery-most-popular')[0];
+
+    mostViewed.manipulationType = 'html';
+    mostViewed.endpoint = '/gallery/most-viewed.json';
+    mostViewed.ready = (): void => {
+        mediator.emit('page:new-content', container);
+    };
```
Comment (SiAdcock) [inline static/src/javascripts/bootstraps/enhanced/gallery.js]: Neither of the `mediator` events are listened for anywhere. Maybe we can delete them and drop the dependency? Or do you reckon consistency with (certain) other bootstraps is more important than dropping dead code?

Comment (sndrs) [reply static/src/javascripts/bootstraps/enhanced/gallery.js]: oh no, good catch, thanks!

Comment (sndrs) [reply static/src/javascripts/bootstraps/enhanced/gallery.js]: in fact nothing seems to listen for `page:new-content`?

Comment (SiAdcock) [reply static/src/javascripts/bootstraps/enhanced/gallery.js]: That's true. And it's pointlessly emitted in 7 places across the application

Comment (sndrs) [reply static/src/javascripts/bootstraps/enhanced/gallery.js]: will remove them in #17940 first. prob worth an event audit I reckon!

**Review by SiAdcock — APPROVED** [review]
Proper tidy

**Pull request merged**